client/state/sites/vouchers/actions.js

```diff
@@ -75,6 +79,68 @@ export const vouchersRequestFailureAction = ( siteId, error ) => {
 	};
 };
 
+export const vouchersAssignReceiveAction = ( siteId, serviceType, voucher ) => {
+	return {
+		type: SITE_VOUCHERS_ASSIGN_RECEIVE,
+		siteId,
+		serviceType,
+		voucher
+	};
+};
+
+/**
+ * Action creator function
+ *
+ * Return SITE_VOUCHERS_ASSIGN_REQUEST action object
```

Comment:
> Personally, if we can't think of a good description for the comment, I take them out altogether because they just get in the way without adding any new knowledge.

I'm going to remove the comments there. There aren't not much to add since these functions are simple.